internal/lint/go.mod

```diff
@@ -0,0 +1,6 @@
+module github.com/Khan/genqlient/internal/lint
+
+go 1.14
```

Comment:
1.14 is *really* old. I kind of think Khan is close to the trailing edge when it comes to Go version support. Most surveys show that 80% are on the current or last version by 6 months after a release. With container based runtimes and dependabot, it's automagical.

Reply:
Yeah, I think that's accurate.  But my general feeling is to wait to drop support until there's a nontrivial reason to, which is why I dropped 1.13 recently, but kept 1.14.